libcaf_net/caf/net/http/v1.cpp

```diff
@@ -30,10 +34,32 @@ writer& operator<<(writer& out, const std::string& str) {
   return out << std::string_view{str};
 }
 
+constexpr std::string_view format(http::method m) {
```

Comment:
Why not use https://github.com/actor-framework/actor-framework/blob/master/libcaf_net/caf/net/http/method.hpp#L55 ?

Reply:
How did I miss this one :dizzy_face: 
I would suggest changing `to_rfc_string` to return `string_view` instead of `string`?

Reply:
I think I've used string for symmetry with `to_string`. However, no strong opinion either way. Feel free to change the return type.